dev-packages/application-package/src/application-props.ts

```diff
@@ -122,6 +123,13 @@ export interface FrontendApplicationConfig extends ApplicationConfig {
      */
     readonly applicationName: string;
 
+    /**
+     * Control if security checks will be bound and executed in your application.
```

Comment:
I was wondering: I am just starting to look at the code, but "security checks" seems a bit generic, no? Anyone who does not look at the code might imagine we do more than check for secure webview deployment? "Secure Webview deployment check", though longer seems more appropriate.

Reply:
Looking further, I see there is more than I thought (preventing opening files in some conditions), so the name might be more appropriate than I thought. It's only in the comments in any case - the preferences have more targeted names